gst_bridge/CMakeLists.txt

```diff
@@ -17,6 +17,12 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic)
 endif()
 
+# Default to Release build
```

Comment:
just quick review: `gst_bridge` sets build type already:

https://github.com/BrettRD/ros-gst-bridge/blob/6ffef792f4a8358d83445f17af8d346f8898146a/gst_bridge/CMakeLists.txt#L4

meaning, the `Release` enforce doesn't take effect here